### PR TITLE
fix: flash screen when creating new page

### PR DESCRIPTION
**Pull request by himself65**

### CODE REVIEW

File: apps/web/src/providers/ThemeProvider.tsx [hunk 1]
```diff
@@ -40,9 +40,9 @@ const ThemeInjector = memo<{
 const ThemeProviderInner = memo<React.PropsWithChildren>(
   function ThemeProviderInner({ children }) {
     const { resolvedTheme: theme } = useTheme();
-    const currentPage = useCurrentPage();
+    const [pageId] = useCurrentPageId();
     const record = useAtomValue(workspacePreferredModeAtom);
-    const editorMode = currentPage ? record[currentPage.id] ?? 'page' : 'page';
+    const editorMode = pageId ? record[pageId] ?? 'page' : 'page';
```
Comment (pengx17) [inline apps/web/src/providers/ThemeProvider.tsx:45]: it looks to me that the mode can also be an atom, but I guess it is not necessary tho

**Review by pengx17 — APPROVED** [review]

File: apps/web/src/providers/ThemeProvider.tsx (hunk 1 above)
Comment (himself65) [reply apps/web/src/providers/ThemeProvider.tsx]: updated

**Pull request merged**